docs/FeatureManagement/FeatureFlag.v2.0.0.schema.sample.json

```diff
@@ -0,0 +1,276 @@
+                "enabled": "true",
+                "conditions": {
+                    "client_filters": [
+                        {
+                            "name": "Microsoft.TimeWindow",
+                            "parameters": {
+                                "Start": "Tue, 27 Jun 2023 06:00:00 GMT"
+                            }
+                        }
+                    ]
+                }
+            },
+            {
+                "id": "Zeta",
+                "description": "A feature flag using a time window filter, that will always return true as the current time is within the time window.",
+                "enabled": "true",
+                "conditions": {
+                    "client_filters": [
+                        {
+                            "name": "Microsoft.TimeWindow",
+                            "parameters": {
+                                "End": "Sat, 28 Jun 3023 06:05:00 GMT"
+                            }
+                        }
+                    ]
+                }
+            },
+            {
+                "id": "Eta",
+                "description": "A feature flag using a targeting filter, that will return true for Alice, Stage1, and 50% of Stage2, and false for Dave and Stage3. The default rollout percentage is 25%.",
+                "enabled": "true",
+                "conditions": {
+                    "client_filters": [
+                        {
+                            "name": "Microsoft.Targeting",
+                            "parameters": {
+                                "Audience": {
+                                    "Users": [
+                                        "Alice"
+                                    ],
+                                    "Groups": [
+                                        {
+                                            "Name": "Stage1",
+                                            "RolloutPercentage": 100
+                                        },
+                                        {
+                                            "Name": "Stage2",
+                                            "RolloutPercentage": 50
+                                        }
+                                    ],
+                                    "DefaultRolloutPercentage": 25,
+                                    "Exclusion": {
+                                        "Users": ["Dave"],
+                                        "Groups": ["Stage3"]
+                                    }
+                                }
+                            }
+                        }
+                    ]
+                }
+            },
+            {
+                "id": "Theta",
+                "description": "A feature flag using a targeting filter, that will return true 74% of the time. Changing to 75% makes the user Brittney true.",
```

Comment:
> Changing to 75% makes the user Brittney true.

We should include another testcase for it.

Reply:
Can't we would need two feature flags with the same name to result in the same bucketing. We could have two different files, but that doesn't really work. My solution for now was the extra section. Maybe with breaking it out into sections like Zhenlan mentioned above it will work out better.